### PR TITLE
Update/tooltip icons classifications TM-1684

### PR DESCRIPTION
**Pull request by scott062**
-Fix spacing on card to center the classification badges
-Update icons
-Add custom codes (all 3 letters) for better readability and identification
-Import FA 5 - Currently runs alongside FA 4 (Need to upgrade across whole site)

### CODE REVIEW

**Comment by scott062** [comment]
Need to consult Sarah about some design decisions. 
1. custom icons (May want cdo feedback) 
2. short codes - remove or update? 
3. Should we continue the colorized scheme or just show applicable badges?

**Comment by mjoyce91** [comment]
Do we have an icon for 6/8?

Screen Shot 2020-03-10 at 9 30 58 AM

File: src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx [hunk 1]
```diff
@@ -1,81 +1,100 @@
...
+import { Tooltip } from 'react-tippy';
+
+import { faLanguage, faGraduationCap, faUserFriends, faMedal, faTree, faRegistered, faChessQueen,
+  faSeedling, faPercentage, faBalanceScaleLeft, faDiceSix } from '@fortawesome/free-solid-svg-icons';
 
 const icons = {
   3: {
     isIcon: true,
-    name: 'bath',
+    name: faSeedling,
```
Comment (mjoyce91) [inline src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx]: 🌱

File: src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx [hunk 2]
```diff
@@ -1,81 +1,100 @@
...
     isIcon: true,
-    name: 'life-ring',
+    name: faRegistered,
     text: 'Recommended for Tenure',
+    shortCode: 'REC',
   },
   6: {
     name: '6/8',
-    text: '6/8 Rule',
+    text: faDiceSix,
```
Comment (mjoyce91) [inline src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx]: 🎲

Comment (scott062) [reply src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx]: Sorry the icon name was under text instead of name

**Comment by mjoyce91** [comment]
Unrelated, but these numbers need to be strings:

```
type: PropTypes.oneOf([
    3,
    4,
    ...
    6,
    ...
```

File: src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx [hunk 3]
```diff
@@ -87,14 +107,21 @@ const ClientBadge = ({ type, status }) => {
...
+        >
+          <FontAwesomeIcon
+            aria-label={ariaLabel}
+            icon={icons[type].name}
+          />
+        </Tooltip>
       </div>
       <div className="client-badge-text">
-        <span>{type}</span>
+        <span>{icons[type].shortCode}</span>
```
Comment (mjoyce91) [inline src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx]: Let's abstract `icons[type].name` to a single variable, and wrap it in a `get`

File: src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx [hunk 4]
```diff
@@ -104,19 +105,20 @@ const status$ = ['none', 'success'];
 const ClientBadge = ({ type, status }) => {
   const isHighlighted = status === true ? 'success' : 'none';
   const ariaLabel = `type of "${type}" with status of "${status$[status]}"`;
+  const icon = get(icons, type, 'None');
   return (
     <div className={`usa-grid-full client-badge-container client-badge-container--${icons[type].isIcon ? 'icon' : 'text'} client-badge-container--${isHighlighted}`}>
```
Comment (mjoyce91) [inline src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx]: Can you add to this line as well?

**Pull request merged**